webpack/components/extensions/HostDetails/DetailsTabCards/ImageModeCard.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { translate as __ } from 'foremanReact/common/I18n';
+import {
+  DescriptionList,
+  DescriptionListGroup,
+  DescriptionListDescription as Dd,
+  DescriptionListTerm as Dt,
+} from '@patternfly/react-core';
+import CardTemplate from 'foremanReact/components/HostDetails/Templates/CardItem/CardTemplate';
+import FontAwesomeImageModeIcon from '../../../../components/extensions/Hosts/FontAwesomeImageModeIcon';
+
+const ImageModeCard = ({ isExpandedGlobal, hostDetails }) => {
+  const imageMode = hostDetails?.content_facet_attributes?.bootc_booted_image;
+  const header = (
+    <>
+      <span style={{ marginRight: '0.5rem' }}>{__('Image mode details')}</span>
+      <FontAwesomeImageModeIcon />
+    </>
+  );
```

Comment:
This could be moved outside the component so it doesn't get recreated on each render. Not essential here though, it shouldn't be too expensive :)